src/components/pages/Profile/Profile.js

```diff
@@ -0,0 +1,62 @@
+/* global gapi */
+import React, { Component } from 'react';
+import {Image} from 'react-bootstrap';
+import './Profile.scss';
+
+class Profile extends Component {
+    constructor() {
+        super();
+
+        this.state = {
+            email: null,
+            profile:  null,
+            profiles: null
+        }
+    }
+
+    
+    loadProfiles(response) {
```

Comment:
This will also need to be stored in the global context? (Redux, React context, etc?)